### PR TITLE
Snapshots in non us-east-1 don't get public

### PR DESCRIPTION
**Pull request by sayanchowdhury**

### CODE REVIEW

File: fedimg/services/ec2.py [hunk 1]
```diff
@@ -717,6 +717,44 @@ def upload(self, compose_meta):
...
+
+                alt_ami = alt_driver.get_image(image.id)
+                blk_device_mapping = alt_ami.extra['block_device_mapping']
+                if len(blk_device_mapping) == 1:
+                    snapshot_id = blk_device_mapping[0]['ebs']['snapshot_id']
+                    snapshot_obj = type('', (), {})()
+                    snapshot_obj.id = snapshot_id
+                    snapshot = alt_driver.list_snapshots(snapshot=snapshot_obj)
+
+                if snapshot is not None:
```
Comment (kushaldas) [inline fedimg/services/ec2.py]:
```
if snapshot:
``` 
This should also do, isn't?

File: fedimg/services/ec2.py [hunk 2]
```diff
@@ -717,6 +717,44 @@ def upload(self, compose_meta):
...
 
+                # Make the snapshot for the image public.
+                is_snapshot_public = False
+                snapshot = None
+
+                alt_ami = alt_driver.get_image(image.id)
+                blk_device_mapping = alt_ami.extra['block_device_mapping']
+                if len(blk_device_mapping) == 1:
+                    snapshot_id = blk_device_mapping[0]['ebs']['snapshot_id']
+                    snapshot_obj = type('', (), {})()
```
Comment (kushaldas) [inline fedimg/services/ec2.py]: Add a comment to explain the next maintainer :)

Comment (sayanchowdhury) [inline fedimg/services/ec2.py]: Ah! right. I had intent to add that.

File: fedimg/services/ec2.py (hunk 1 above)
Comment (sayanchowdhury) [inline fedimg/services/ec2.py]: [From the horse's mouth](https://www.python.org/dev/peps/pep-0008/#programming-recommendations)

Comparisons to singletons like None should always be done with is or is not , never the equality operators.

Also, beware of writing if x when you really mean if x is not None -- e.g. when testing whether a variable or argument that defaults to None was set to some other value. The other value might have a type (such as a container) that could be false in a boolean context!

**Comment by sayanchowdhury** [comment]
Thanks @kushaldas for the review.

**Pull request merged**